src/shared/GlobalTopBanners/OktaEnabledBanner/OktaEnforcedBanner/OktaEnabledBanner.tsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from 'react-router-dom'
+
+import Button from 'ui/Button'
+import Icon from 'ui/Icon'
+import TopBanner from 'ui/TopBanner'
+
+interface URLParams {
+  owner: string
+}
+
+const OktaEnabledBanner = () => {
+  const { owner } = useParams<URLParams>()
```

Comment:
Will need to swap `owner` to be optional here, because `GlobalTopBanners` is used in places where owner is not defined. If it's not defined the component should probably just early return with `null`.